OptimizelySDK/Odp/OdpEventManager.cs

```diff
@@ -90,16 +103,29 @@ private void DropQueue()
             }
         }
 
+        /// <summary>
+        /// Begin the execution thread to process the queue into bathes and send events
+        /// </summary>
         public void Start()
         {
-            if ((IsStarted && !Disposed) || !_odpConfig.IsReady())
+            if (!_odpConfig.IsReady())
             {
                 _logger.Log(LogLevel.WARN, Constants.ODP_NOT_INTEGRATED_MESSAGE);
 
                 DropQueue();
 
                 return;
             }
+
+            if (IsStarted && !Disposed)
+            {
+                _logger.Log(LogLevel.WARN, Constants.ODP_NOT_ENABLED_MESSAGE);
```

Comment:
Is this message correct for when IsStarted is already ON?

Reply:
I agree with you.  It doesn't seem quite right.  

I couldn't decide which message to provide from the TDD's "3.6 Errors/Log Messages".

I was hoping for something like "ODP has not been started."

Reply:
@mikechu-optimizely It should be something like "already started". Not related to ODP.